### PR TITLE
Add 2.11.4 Release notes

### PR DESCRIPTION
**Pull request by billpalombi**
Need to add section on `flow.visualize()`

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
...
+
+### Guide to upgrade from agents to workers
+Upgrading to workers significantly enhances the experience of deploying flows. It simplifies the specification of each flow's infrastructure and runtime environment. 
+
+A [worker](/concepts/work-pools/#worker-overview) is the fusion of an [agent](/concepts/agents/) with an [infrastructure block](/concepts/infrastructure/). Like agents, workers poll a work pool for flow runs that are scheduled to start. Like infrastructure blocks, workers are typed - they work with only one kind of infrastructure and they specify the default configuration for jobs submitted to that infrastructure.
+
+We've written [a handy guide](https://github.com/PrefectHQ/prefect/pull/10365) that describes how to upgrade from agents to workers in just a few quick steps.
+
+### Visualize your flow before running it
+
```
Comment (WillRaphaelson) [inline RELEASE-NOTES.md:13]:
```suggestion
You can now get a quick sense of the structure of your flow using the `.visualize()` method on your flow. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code. - https://github.com/PrefectHQ/prefect/pull/10417
```

Comment (billpalombi) [reply RELEASE-NOTES.md:19]: Added it

**Review by pleek91 — APPROVED** [review]
LGTM

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
 # Prefect Release Notes
 
+## Release 2.11.4
+
+### Guide to upgrade from agents to workers
+Upgrading to workers significantly enhances the experience of deploying flows. It simplifies the specification of each flow's infrastructure and runtime environment. 
+
+A [worker](/concepts/work-pools/#worker-overview) is the fusion of an [agent](/concepts/agents/) with an [infrastructure block](/concepts/infrastructure/). Like agents, workers poll a work pool for flow runs that are scheduled to start. Like infrastructure blocks, workers are typed - they work with only one kind of infrastructure and they specify the default configuration for jobs submitted to that infrastructure.
+
+We've written [a handy guide](https://github.com/PrefectHQ/prefect/pull/10365) that describes how to upgrade from agents to workers in just a few quick steps.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:10]: should we be linking to the PR or linking to the docs?

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
+
+### Visualize your flow before running it
+Until now, the only way to produce a visual schematic of your flow has been to run it. Some flows, though, are time consuming or expensive to run. Now, you can get a quick sense of the structure of your flow using the `.visualize()` method. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:13]:
```suggestion
Until now, the only way to produce a visual schematic of your flow has been to run it. Some flows, though, are time consuming or expensive to run. Now, you can get a quick sense of the structure of your flow using the `.visualize()` method. Calling this method will attempt to produce a schematic diagram of your flow and tasks without actually running your flow code.
```

would it be good to mention that it was only viewable in the Prefect UI but is now rendered locally as a png?

File: RELEASE-NOTES.md (hunk 2 above)
Comment (billpalombi) [reply RELEASE-NOTES.md]: Good catch!

File: RELEASE-NOTES.md (hunk 3 above)
Comment (billpalombi) [reply RELEASE-NOTES.md]: I like that.

**Pull request merged**